src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntityChangeValidator.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation;
+
+import org.apache.commons.lang.StringUtils;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.model.company.RegisteredOfficeAddressApi;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.AddressDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.changes.*;
+
+@Component
+public class OverseasEntityChangeValidator {
+    public EntityNameChange verifyEntityNameChange(String existing, String updated) {
```

Comment:
For all changes. If proposed is null then no change. If current is null then a change if proposed is not null.